modules/comment-likes/admin-style.css

```diff
@@ -0,0 +1,55 @@
+.vers img {
```

Comment:
Since the image that it's styling is not needed, I think we can just remove this block too.